integrations-and-supported-tools/great-expectations/notebooks/Neptune_Great_Expectations.ipynb

```diff
@@ -0,0 +1,550 @@
+    "\n",
+    "If you want to see the example logged to your own workspace instead:\n",
+    "\n",
+    "  1. Create a Neptune account. [Register &rarr;](https://neptune.ai/register)\n",
+    "  1. Create a Neptune project that you will use for tracking metadata. For instructions, see [Creating a project](https://docs.neptune.ai/setup/creating_project) in the Neptune docs."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "NTVvWzR-eluS"
+   },
+   "source": [
+    "## Install Neptune and GX OSS"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "7S4oP2QneluT",
+    "tags": [
+     "installation"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "%pip install -q -U neptune great_expectations"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "I9FI9oaqeluU",
+    "tags": [
+     "header"
+    ]
+   },
+   "source": [
+    "## Import libraries"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "0CrsGX17eluV",
+    "tags": [
+     "code"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "import neptune\n",
+    "import great_expectations as gx\n",
+    "\n",
+    "from neptune.utils import stringify_unsupported"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Start a Neptune run\n",
+    "\n",
+    "To create a new run for tracking the metadata, you tell Neptune who you are (`api_token`) and where to send the data (`project`).\n",
+    "\n",
+    "You can use the default code cell below to create an anonymous run in the public project [common/great-expectations](https://app.neptune.ai/o/common/org/great-expectations).\n",
+    "\n",
+    "**Note**: Public projects are cleaned regularly, so anonymous runs are only stored temporarily.\n",
+    "\n",
+    "### Log to your own project instead\n",
+    "\n",
+    "Replace the code below with the following:\n",
+    "\n",
+    "```python\n",
+    "import neptune\n",
+    "from getpass import getpass\n",
+    "\n",
+    "run = neptune.init_run(\n",
+    "    project=\"workspace-name/project-name\",  # Replace with your workspace and project names\n",
+    "    api_token=getpass(\"Enter your Neptune API token: \"),\n",
+    "    tags=[\"notebook\"],  # (Optional) Replace with your own tags\n",
+    ")\n",
+    "```\n",
+    "\n",
+    "To find your API token and full project name:\n",
+    "\n",
+    "1. [Log in to Neptune](https://app.neptune.ai/).\n",
+    "1. In the bottom-left corner, expand your user menu and select **Get your API token**.\n",
+    "1. The workspace name is displayed in the top-left corner of the app.\n",
+    "\n",
+    "    To copy the project path, in the top-right corner, open the settings menu and select **Properties**.\n",
```

Comment:
This is from the old UI.

Let's be so bold as to future-proof it beyond tomorrow :D

```suggestion
    "    To copy the project path, in the top-right corner, open the settings menu and select **Details & privacy**.\n",
```

Reply:
This and the below suggestion (https://github.com/neptune-ai/examples/pull/328/files#r1651006639) are on the same line. Which one do I apply?